taxonomy/models.py

```diff
@@ -0,0 +1,31 @@
+"""Models for lore taxonomy application"""
+
+from django.db import models
+
+
+class Vocabulary(models.Model):
+    repository = models.ForeignKey("Repository")
+    name = models.TextField()
```

Comment:
To prevent `null` values in the database, if this field isn't required it should have `blank=True, default=""` in the definition. Also, I believe a `TextField` requires a `max_length` value.